src/style/App.css

```diff
@@ -1,106 +1,247 @@
 @import url("https://fonts.googleapis.com/css?family=Roboto:400,500");
-html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, hr, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
```

Comment:
It looks like Prettier attacked this file in an attempt to make it tidier, but has left it worse.

Would you rather I keep it as before?